application/src/main/java/org/togetherjava/tjbot/commands/filesharing/package-info.java

```diff
@@ -0,0 +1,5 @@
+/**
+ * This package offers all the functionality for automatically uploading files to gist The core
```

Comment:
dont specify the provider here, that might drift apart from the actual implementation. `for automatically uploading files to sharing services`. also, missing `.` (fullstop) at the end of the sentence.